tests/integration_tests/integration_test.py

```diff
@@ -129,15 +148,17 @@ def get_jira_client():
     :return: JIRA Client
     :rtype: jira.client.JIRA
     """
-    return jira.client.JIRA(**{
-        'options': {
-            'server': URL,
-            'verify': False,
-        },
-        'token_auth': TOKEN,
-    })
-
-
-if __name__ == '__main__':
+    return jira.client.JIRA(
+        **{
+            "options": {
+                "server": URL,
+                "verify": False,
+            },
+            "token_auth": TOKEN,
+        }
+    )
```

Comment:
Using the `**` operator on a literal is a little heavy-handed, we should do something like this, instead:
```suggestion
    return jira.client.JIRA(
        options={
            "server": URL,
            "verify": False,
        },
        token_auth=TOKEN,
    )
```
_[Again, feel free to decline if you aren't interested in this kind of change in this PR.]_